x-pack/plugins/upgrade_assistant/server/lib/reindexing/worker.ts

```diff
@@ -60,10 +84,6 @@ export class ReindexWorker {
     this.log = log.get('reindex_worker');
     this.security = security;
 
-    if (ReindexWorker.workerSingleton) {
-      throw new Error(`More than one ReindexWorker cannot be created.`);
-    }
```

Comment:
I had to change this to log a warning instead of throwing an error in order to start multiple Kibana instances in memory. Fun fact: as I was looking at this and thinking "who in the world would do this?" I learned that, in fact, I did wrote this code 3 years ago 😄